### PR TITLE
chore(signpost): style documentation

### PR DESCRIPTION
**Pull request by valentin-mladenov**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/angular/src/popover/signpost/STYLES.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Combobox
```
Comment (dtsanevmw) [inline projects/angular/src/popover/signpost/STYLES.md:1]:
```suggestion
# SIGNPOST - Style API
```

File: projects/angular/src/popover/signpost/STYLES.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## CSS Custom Properties
+
+| CSS Variable Name                       | Description                        |
```
Comment (dtsanevmw) [inline projects/angular/src/popover/signpost/STYLES.md:5]:
```suggestion
| CSS Custom Property                       | Description                        |
```

**Review by dtsanevmw — CHANGES_REQUESTED** [review]
Looks good, only two small typos.

File: projects/angular/src/popover/signpost/STYLES.md (hunk 2 above)
Comment (valentin-mladenov) [reply projects/angular/src/popover/signpost/STYLES.md]: corrected in commit be0b983

**Review by dtsanevmw — APPROVED** [review]

**Pull request merged**